src/gfx.cpp

```diff
@@ -1113,14 +1113,14 @@ void DoPaletteAnimations()
  * @param background Background colour.
  * @return TC_BLACK or TC_WHITE depending on what gives a better contrast.
  */
-TextColour GetContrastColour(uint8 background)
+TextColour GetContrastColour(uint8 background, uint8 threshold)
```

Comment:
doc comments should be extended accordingly

Reply:
Thanks, this should be fixed in the most recent update of the branch.